### PR TITLE
Validation home page

### PR DESCRIPTION
**Pull request by aminalakhsham**
Relates #33  
add validation for routes /register and /contact-us.

### CODE REVIEW

File: public/js/user/home.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+const formContactUs = document.querySelector('.footer--contactUs-form');
+const registerEmail = document.querySelector('.footer--form');
```
Comment (InassTubail) [inline public/js/user/home.js]: const formContactUs = document.querySelector('.footer--contactUs-form');
const registerEmail = document.querySelector('.footer--form'); 

make your code more abstract.
This method "Document.querySelector()"
You will use it frequently in more than one place, so you can write it in another file then call it anywhere.
As => const select = (selector) => document.querySelector(selector);

But As you prefer   :+1:

File: public/js/user/home.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  const dataregisterEmail = {};
+  formData.forEach((value, key) => {
+    dataregisterEmail[key] = value;
+  });
+  fetch('/register', {
+    method: 'POST',
+    body: JSON.stringify(dataregisterEmail),
+    headers: { 'Content-Type': 'application/json' },
+  })
+    .then(res => console.log(res.json()));
```
Comment (InassTubail) [inline public/js/user/home.js]: Remove console logs.

**Review by InassTubail — APPROVED** [review]

File: public/js/user/home.js (hunk 1 above)
Comment (aminalakhsham) [reply public/js/user/home.js]: ok

File: public/js/user/home.js (hunk 2 above)
Comment (aminalakhsham) [reply public/js/user/home.js]: i'll do that after database done cuz no error show while i'm do the dom and css files

File: public/js/user/home.js (hunk 1 above)
Comment (aminalakhsham) [reply public/js/user/home.js]: i'm add function for it
you can see it

Comment (aminalakhsham) [reply public/js/user/home.js]: @InassTubail

**Review by InassTubail — APPROVED** [review]

**Pull request merged**